stable/atlantis/values.yaml

```diff
@@ -68,6 +68,18 @@ serviceAccountSecrets:
   # credentials: <json file as base64 encoded string>
   # credentials-staging: <json file as base64 encoded string>
 
+# To specify Azure credentials to be mapped as env vars if deploying to Azure cloud
```

Comment:
These values should use camel case per the helm naming conventions.
https://helm.sh/docs/chart_best_practices/#naming-conventions

Reply:
This was addressed.